pisa/stages/data/grid.py

```diff
@@ -0,0 +1,107 @@
+"""
```

Comment:
Is this any different from using the `pisa/stages/data/toy_event_generator.py` with mode `binned` and `random=False`?
Should check if we can remove code duplication here

Reply:
Using `toy_event_generator.py` in binned mode just produces a binned version of the generated data (which follows a power law spectrum) I believe. `grid.py` produces an actual grid, e.g. one event per grid point. There are also complications with bin edges vs bin centers I believe.

Reply:
So I think these two provide different behaviour right now. `grid.py` is good for plotting flux, oscillograms, etc.

Reply:
ah yeah, I think you're right

Reply:
I have a feeling that maybe `toy_event_generator.py` did produce grid-like outputs in the past but no longer does, but maybe I'm imagining that. In any case I think these are two distinct and both valid cases.

Reply:
Did notice that the docs were out of date though in this stage, have updated now.